script/Arcueid/Day 13/13_00_ARC13_6.txt

```diff
@@ -446,8 +447,8 @@ They remind me a lot of what that woman said to me,
 [sha:678158c8a1d4b29a157a299c8e2dd8c23a8086a9]{
 -- Page 476, Offset 33048.
 -- 「……そう、だね。……なんとなく―――どこか救いが、残っているのかも、しれないなら」
-"...I guess you're right.
- I feel like―――there might still be a chance at salvation out there yet."
+"I guess you're right...
+ I feel like――there might still be a chance at salvation out there yet."
```

Comment:
I feel like > Someway, somehow

and then the dash has justified its existence.